components/TaskDetails.js

```diff
@@ -2,22 +2,30 @@
 import React from 'react'
 
 // React Native
-import { View, Text, StyleSheet } from 'react-native'
+import { View, Text, Picker, StyleSheet } from 'react-native'
 import { Icon } from 'react-native-elements'
 
 // Moment
 import moment from 'moment'
 
 export default class TaskDetails extends React.Component {
+  state = {status: ''}
+  updateStatus = (status) => { this.setState({status: status}) }
   render () {
     const task = this.props.task
     return (
       <View style={styles.container}>
         <View >
           <View style={styles.media}>
             <Text style={styles.taskId}> #{task.id} </Text>
-            <Text style={styles.label}> { task.status.title } </Text>
+            <Picker selectedValue={this.state.status}
+              onValueChange= {this.updateStatus} style={{ height: 200, width: 120}}>
+              <Picker.Item label ="Created" value="Created" />
+              <Picker.Item label ="InProgress" value="InProgress" />
```

Comment:
May be we can use loop here, by passing the values to an array. It is just to make code more nearer to actual api implementation.